ui/app/pages/token/index.scss

```diff
@@ -1,3 +1,152 @@
 @import 'fee-card/index';
 @import 'exchange-rate-display/index';
 @import 'actionable-message/index';
+
+.token {
+  &__search-token {
+    margin: 24px;
+    border: 1px solid #fff;
+    box-sizing: border-box;
+    box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
+    border-radius: 6px;
+    min-height: 297px;
+    width: 100%;
+
+    .searchable-item-list {
+      &__search {
+        div {
+          border: none;
+          border-bottom: 1px solid #d2d8dd;
+          border-radius: 0;
+          height: 61px;
+
+          input {
+            @include H4;
+
+            font-weight: bold;
+            color: $Black-100;
+          }
+        }
+      }
+
+      &__list-container {
+        display: flex;
+        flex-direction: column;
+        max-height: 330px;
+        overflow-y: scroll;
+        scrollbar-width: none;
+
+        &::-webkit-scrollbar {
+          display: none;
```

Comment:
Given the recent confusion we've seen around hidden scrollbars, I don't think this is something we should do anymore in general (e.g. https://consensys.slack.com/archives/G4V2HTG0Y/p1595014237319800)